src/Codeception/Module/Symfony.php

```diff
@@ -521,9 +521,15 @@ protected function debugResponse($url)
      */
     private function extractRawRoles(Data $data)
     {
-        $raw = $data->getRawData();
+        // prior to Symfony 3.3
```

Comment:
Are you sure that this comment is correct?
`getRawData` method is deprecated since 3.3 and [removed in 4.0](https://github.com/symfony/var-dumper/commit/18e4a4c30f10f5879129cbcf73c4a5b9af745abc).